GIS.md

```diff
@@ -0,0 +1,175 @@
+# Why Gnome-Initial-Setup
+There are several use cases where it makes sense for system installation and user creation to not happen at the same time. In these instances, it is useful to have a way to skip the user creation step during installation and for it to be handled later, perhaps even by a different user.
```

Comment:
We do a max line-length of 120 in the readme I think, maybe we should do the same here so that the raw file is easier to read?

Reply:
Ah good point, yes, let me amend that quick :smile:

Reply:
@spydon Out of interest, does that line limit apply hyperlinks?

Reply:
Usually no :)

Reply:
When they are in the end of the line that is